ovn-tester/tests/service_route.py

```diff
@@ -0,0 +1,82 @@
+from collections import namedtuple
+from ovn_context import Context
+from ovn_workload import Namespace
+from ovn_ext_cmd import ExtCmd
+
+import netaddr
+import ovn_load_balancer as lb
+
+ServiceRouteCfg = namedtuple('ServiceRouteCfg',
+                             ['n_lb_cluster',
+                              'n_lb_node'])
```

Comment:
I would keep it simple and use a single `n_lb` argument.

For every LB add a cluster-wide load balancer and N node-specific load balancers where N is the number of worker nodes in the cluster.  Wdyt?

Reply:
@dceara ack, I will fix it

Reply:
I added some more comments on the newest version.